Xamarin.Forms.Core/FluentExtensions.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+
+namespace Xamarin.Forms.Fluent
+{
+	public static class FluentExtensions
+	{
+		#region BindableObject section
+
+		public static TView Fluent<TView>(this TView self,
```

Comment:
there is very little gain to be had over here, I'd remove this call

Reply:
@StephaneDelcroix as i mentioned above. It can be useful for event subscriptions..